src/CMakeLists.txt

```diff
@@ -35,3 +35,12 @@ endif()
 
 add_library(cucumber-cpp-nomain STATIC ${CUKE_SOURCES})
 add_library(cucumber-cpp STATIC ${CUKE_SOURCES} main.cpp)
+
+if(WIN32 AND NOT MSVC)
+    target_link_libraries(cucumber-cpp-nomain
+        ws2_32
+        )
+    target_link_libraries(cucumber-cpp
+        ws2_32
+        )
+endif(WIN32 AND NOT MSVC)
```

Comment:
Actually I don't know if other compilers need this library as well. i only have MinGW and MSVC available on my windows machine. I know that MinGW requires it, but MSVC doesn't. However I think it would also work when the library is added on MSVC also. I will check that.